### PR TITLE
Fixed the 'from config' overrides

### PR DESCRIPTION
**Pull request by fhamon**

### CODE REVIEW

**Comment by fhamon** [comment]
Forgot to make my changes in a new branch. Sorré.

File: email-gateways/email.mailjet.php [hunk 1]
```diff
@@ -15,43 +15,42 @@ public function about()
...
+                    'Name' => $this->_sender_name
                 ],
                 'To' => [[
                     'Email' => $address,
                     'Name' => is_numeric($name) ? $address : $name,
                 ]],
                 'Subject' => $this->_subject,
-                'TextPart' => $this->_text_plain,
-                'HTMLPart' => $this->_text_html,
+                'TextPart' => empty($this->_text_plain) ? strip_tags($html) : $this->_text_plain,
```
Comment (nitriques) [inline email-gateways/email.mailjet.php]: `strip_tags($html)` should be `strip_tags($this->_text_html)`

File: email-gateways/email.mailjet.php [hunk 2]
```diff
@@ -15,43 +15,42 @@ public function about()
...
+                'HTMLPart' => empty($this->_text_html) ? '' : $this->_text_plain,
```
Comment (nitriques) [inline email-gateways/email.mailjet.php]: should be `empty($this->_text_html) ? '' : $this->_text_html`

**Review by nitriques — CHANGES_REQUESTED** [review]

File: email-gateways/email.mailjet.php (hunk 1 above)
Comment (fhamon) [reply email-gateways/email.mailjet.php]: nice catch 👍

File: email-gateways/email.mailjet.php (hunk 2 above)
Comment (fhamon) [reply email-gateways/email.mailjet.php]: 👍

**Review by nitriques — APPROVED** [review]

**Pull request merged**